assets/js/membraneWebRTC.ts

```diff
@@ -105,6 +107,10 @@ export interface Callbacks {
    * Called in case of errors related to multimedia session e.g. ICE connection.
    */
   onConnectionError?: (message: string) => void;
+  /**
+   * Called each time the peer which was already in the room, add new track.
```

Comment:
```suggestion
   * Called each time the peer which was already in the room, adds new tracks.
```
Also, shouldn't this callback take something more than Peer?